ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSemanticAnalyzer.java

```diff
@@ -299,18 +184,6 @@ private void analyzeReplDump(ASTNode ast) throws SemanticException {
     }
   }
 
-  private void dumpEvent(NotificationEvent ev, Path evRoot, Path cmRoot) throws Exception {
-    EventHandler.Context context = new EventHandler.Context(
-        evRoot,
-        cmRoot,
-        db,
-        conf,
-        getNewEventOnlyReplicationSpec(ev.getEventId())
-    );
-    EventHandlerFactory.handlerFor(ev).handle(context);
-    REPL_STATE_LOG.info("Repl Dump: Dumped event with ID: {}, Type: {} and dumped metadata and data to path {}",
-                        String.valueOf(ev.getEventId()), ev.getEventType(), evRoot.toUri().toString());
-  }
 
   public static void injectNextDumpDirForTest(String dumpdir){
```

Comment:
Remove the unused methods such as removeValuesTemporaryTables, matchesTbl, matchesDb, getNewEventOnlyReplicationSpec, getNewReplicationSpec from ReplicationSemanticAnalyzer.java.

Also, update removeTemporaryTablesForMetadataDump test method to refer ReplDumpWork class.